experimental/dds/tree2/src/util/brand.ts

```diff
@@ -29,7 +76,7 @@ export type Brand<ValueType, Name extends string> = ValueType & BrandedType<Valu
  * - get nominal typing (so types produced without using this class can never be assignable to it).
  * - allow use as {@link Opaque} branded type (not assignable to `ValueType`, but captures `ValueType`).
  *
- * See {@link InternalTypes#MakeNominal} for some more details.
+ * See `InternalTypes.MakeNominal` for some more details.
```

Comment:
This used to be a bad link, just in a format API-Extractor didn't validate (external package link). There isn't a way to fix the link as far as I know since using "." to traverse namespaces leads to:
`@fluid-experimental/tree2: Error: src/util/brand.ts:92:1 - (ae-unresolved-link) The @link reference could not be resolved: This type of declaration is not supported yet by the resolver` so this has been unlinked.